line-bot-webhook/src/test/resources/callback/location.json

```diff
@@ -0,0 +1,23 @@
+{
+  "destination": "Uab012345678901234567890123456789",
+  "events": [
+    {
+      "replyToken": "nHuyWiB7yP5Zw52FIkcQobQuGDXCTA",
+      "type": "message",
+      "timestamp": 1462629479859,
+      "mode": "active",
+      "source": {
+        "type": "user",
+        "userId": "u206d25c2ea6bd87c17655609a1c37cb8"
+      },
+      "message": {
+        "id": "325708",
+        "type": "location",
+        "title": "hello",
+        "address": "〒150-0002 東京都渋谷区渋谷２丁目２１−１",
```

Comment:
nit: this address is too old...?

Reply:
Ya, it's a old LINE's office address, but we don't need to update here :P
I don't want to update it every time we move our office. hehe

Reply:
😄